predictsignauxfaibles/data.py

```diff
@@ -204,6 +204,20 @@ def remove_na(self, ignore: list):
         self.data.reset_index(drop=True, inplace=True)
         return self
 
+    def remove_siren(self, siren_list: list):
+        """
+        Removes all observations corresponding to
+        one of the siren provided as input.
+        Args:
+            siren_list: a list of siren to be removed from the data.
+        """
+        orig_length = len(self.data.index)
```

Comment:
you can use `len(self)`

Reply:
My bad, didn't see method len() specialised here